templates/themes/app/backoffice/layout.html.twig

```diff
@@ -0,0 +1,103 @@
+{% trans_default_domain 'admin' %}
+
+{% set user_initials = ibexa_field_value(app.user.APIUser, 'first_name').text[0:1] ~ ibexa_field_value(app.user.APIUser, 'last_name').text[0:1] %}
+
+<!DOCTYPE html>
+<html lang="{{ app.request.locale|replace({'_': '-'}) }}">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+    <meta name="theme-color" content="#ffffff">
+
+    {% block meta %}{% endblock %}
+
+    {% for href in encore_entry_css_files('backoffice-index', 'app') %}
+        <link rel="stylesheet" type="text/css" href="{{ href }}">
+    {% endfor %}
+
+    {% block stylesheets %}{% endblock %}
+
+    <script src="{{ asset('bundles/netgenibexafieldtypehtmltext/build/app.js') }}"></script>
```

Comment:
This should be removed, it's not used or installed.

Reply:
@pspanja removed https://github.com/netgen/media-site/commit/2341a964c1414464f39cd7275a4fb744a3434507